### PR TITLE
global: services removal and fix wheel configuration setup

### LINKED ISSUE #17: services: remove services from master
#### Question

Are the services used or will be deprecated and substituted with something new?

I'm talking about: [services.py](https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/services.py) and all [services](https://github.com/inveniosoftware/invenio-search/tree/master/invenio_search/searchext/services).

##### Issues correlated:
- It depends from [knowledge](https://github.com/inveniosoftware/invenio-search/issues/13).
- Also the [Legacy removal](https://github.com/inveniosoftware/invenio/issues/3233) issue because most of the imports are from legacy code.

@jirikuncar @egabancho @kaplun @tiborsimko

### PR DESCRIPTION
**Pull request by hachreak**
* INCOMPATIBLE Removes search service plugins that are going to be
  re-implemented using the new API. (closes #17)
  (addresses inveniosoftware/invenio#3233) (addresses #13)

* Fixes wheel configuration setup.

* fix dependency configuration.

### CODE REVIEW

File: .travis.invenio.cfg [hunk 1]
```diff
@@ -30,11 +30,14 @@ SECRET_KEY = 'MY_SECRET'
 ASSETS_AUTO_BUILD = False
 
 PACKAGES = [
-    'invenio_search',
-    'invenio_knowledge',
     'invenio_access',
     'invenio_accounts',
+    'invenio_base',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: the order is important ... invenio_base should be "last"

File: setup.py [hunk 2]
```diff
@@ -35,14 +35,16 @@
 
 requirements = [
     'Flask>=0.10.1',
-    'six>=1.7.2',
     'invenio-access>=0.1.0',
     'invenio-accounts>=0.1.2',
     'invenio-base>=0.1.0',
+    'invenio-collections>=0.1.2',
```
Comment (jirikuncar) [inline setup.py]: please remove this dependency ...

File: requirements-devel.txt [hunk 3]
```diff
@@ -25,7 +25,9 @@
 -e git+git://github.com/inveniosoftware/invenio-access.git#egg=invenio-access
 -e git+git://github.com/inveniosoftware/invenio-accounts.git#egg=invenio-accounts
 -e git+git://github.com/inveniosoftware/invenio-base.git#egg=invenio-base
+-e git+git://github.com/inveniosoftware/invenio-collections.git#egg=invenio-collections
```
Comment (jirikuncar) [inline requirements-devel.txt]: please remove invenio-collections

File: setup.py (hunk 2 above)
Comment (jirikuncar) [inline setup.py]: you might include it only for tests

Comment (hachreak) [inline setup.py]: Only in tests?
Currently invenio-collections is imported by:
 - invenio_search/admin.py 
 - invenio_search/enhancers/collection_filter.py
 - invenio_search/facet_builders.py
 - invenio_search/forms.py
 - invenio_search/registry.py
 - invenio_search/utils.py
 - invenio_search/views/search.py

Comment (jirikuncar) [inline setup.py]: I know it's not correct but we need to fix those files.

File: .travis.invenio.cfg [hunk 4]
```diff
@@ -30,11 +30,14 @@ SECRET_KEY = 'MY_SECRET'
...
 
 PACKAGES = [
-    'invenio_search',
-    'invenio_knowledge',
     'invenio_access',
     'invenio_accounts',
+    'invenio_collections',
     'invenio_formatter',
+    'invenio_knowledge',
+    'invenio_query_parser',
```
Comment (jirikuncar) [inline .travis.invenio.cfg]: this one should not be registered in `PACKAGES`

File: .travis.yml [hunk 5]
```diff
@@ -54,8 +54,8 @@ before_install:
   - "python requirements.py --extras=$REXTRAS --level=min > .travis-lowest-requirements.txt"
   - "python requirements.py --extras=$REXTRAS --level=pypi > .travis-release-requirements.txt"
   - "python requirements.py --extras=$REXTRAS --level=dev > .travis-devel-requirements.txt"
-  - "mkdir -p ${VIRTUAL_ENV}/var/invenio.base-instance/"
-  - "cp .travis.invenio.cfg ${VIRTUAL_ENV}/var/invenio.base-instance/invenio.cfg"
+  - "mkdir -p ${VIRTUAL_ENV}/var/invenio_base-instance/"
```
Comment (jirikuncar) [inline .travis.yml]: keep `invenio.base`

**Pull request merged**